Reemit.Decompiler.Clr/Metadata/Streams/MetadataTablesStream.cs

```diff
@@ -52,6 +54,8 @@ public MetadataTablesStream(BinaryReader reader)
         TypeRef = ReadTableIfExists<TypeRefRow>(MetadataTableName.TypeRef);
         TypeDef = ReadTableIfExists<TypeDefRow>(MetadataTableName.TypeDef);
         Field = ReadTableIfExists<FieldRow>(MetadataTableName.Field);
+        ClassLayout = ReadTableIfExists<ClassLayoutRow>(MetadataTableName.ClassLayout);
```

Comment:
According to [this blog post](https://www.red-gate.com/simple-talk/blogs/anatomy-of-a-net-assembly-clr-metadata-3/), table bytes are ordered by the table number. `ClassLayout` has table number 0x0F, which puts it right after `DeclSecurity`. It seems that we might have to implement reading the following tables before we can read `ClassLayout`:
```
MethodDef
Param
InterfaceImpl
MemberRef
Constant
CustomAttribute
FieldMarshal
DeclSecurity
```

I suggest commenting out this line and the line below it for now. At least until we implement tables preceding these.

Reply:
Ahhhh, okay. Wondered why I was getting junk, that makes sense.